move-if-change

```diff
@@ -1,15 +0,0 @@
-#!/bin/sh
```

Comment:
I have no idea why this script is needed. It's not used anywhere in the HDF4 source.